tests/integration_tests/utils.py

```diff
@@ -351,4 +351,4 @@ def cb(attrs):
     proposal = cli.query_proposal(proposal_id)
     wait_for_block_time(cli, isoparse(proposal["voting_end_time"]))
     proposal = cli.query_proposal(proposal_id)
-    assert proposal["status"] == status, proposal
+    assert proposal["status"] == "PROPOSAL_STATUS_PASSED", proposal
```

Comment:
we need keep both  number and string enum check right?

Reply:
why?